contrib/ci/Dockerfile-fedora.in

```diff
@@ -3,6 +3,7 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 RUN dnf --enablerepo=updates-testing -y update
+RUN echo fubar > /etc/machine-id
```

Comment:
Since you were echoing this for both arch and Fedora, was that because they were empty there?  Or you are trying to make sure it's the same result every time?

If so, probably need to add to the other Dockerfiles for Debian and such too.

Reply:
It was just to replace the empty value, which is now an error. Do you think the other Dockerfiles should have the same?

Reply:
I suppose if there is ever going to be a full CI test run that includes reporting up to LVFS they should all be the same (so you can filter CI based uploads).

Reply:
That's a very good point. I'll fix this based on that logic alone. Is the RUN in the dockerfile the best place to set the machine-id?

Reply:
Yep I think so.

Reply:
Fixed, thanks.